test/cypress/integration/04-collective_v2.test.js

```diff
@@ -1,16 +1,10 @@
 import { disableSmoothScroll } from '../support/helpers';
 import { Sections } from '../../../components/collective-page/_constants';
+import mockRecaptcha from '../mocks/recaptcha';
 
 describe('New collective page', () => {
   let collectiveSlug = null;
 
-  const scrollToSection = section => {
-    // Wait for new collective page to load before disabling smooth scroll
-    cy.contains('Become a financial contributor');
-    disableSmoothScroll();
-    cy.get(`#section-${section}`).scrollIntoView();
-  };
-
```

Comment:
Why moving this?

Reply:
I didn't feel that the tests I added where part of the describe `New collective page`. The current tests under this describe are like: 1) create the collective, 2) go to the collective, 3) check that what you are testing is there.

In my case I wanted to do somethings before going to the collective to check, and I wanted to use the function `scrollToSection` in my test too.

Reply:
I moved it to the top.